src/components/sbb-navigation/sbb-navigation.tsx

```diff
@@ -377,6 +377,7 @@ export class SbbNavigation implements ComponentInterface {
             aria-label={this.accessibilityLabel}
             onAnimationEnd={(event: AnimationEvent) => this._onAnimationEnd(event)}
             class="sbb-navigation"
+            role="group"
```

Comment:
Is this `group` due to `aria-label`?

Reply:
Yes, as it reads the aria label, which it doesn't when it don't have a specific role. as group is very neutral, we decided to go with it